deploy/jenkins-ci/Jenkinsfile

```diff
@@ -195,7 +179,10 @@ node('kiali-build && fedora') {
         returnStdout: true,
         script: "${backendDir}/deploy/jenkins-ci/bin/determine-release-type.sh").trim()
     }
+
+    buildSite = params.SKIP_SITE_RELEASE != "y" && releaseType == "minor"
```

Comment:
minor: I would move this up on the website, ideally you should be able to skim there and understand at least the rules for release.

Reply:
I'm not sure what you meant for "moving up" :-/
Anyway, this is a partial PR. There is still more work for #2895. So I'll merge and fix in a follow-up when you clarify ;)